### PR TITLE
New Tests for sets module

### PR DESCRIPTION
**Pull request by vijairamg**
References to other Issues or PRs

Related to #16318 (Increase test coverage)

Brief description of what is fixed or changed

* Added new tests for conditionset.py , contains.py , ordinals.py and
  powerset.py

NO ENTRY

### CODE REVIEW

File: sympy/sets/tests/test_conditionset.py [hunk 1]
```diff
@@ -267,3 +267,9 @@ def test_flatten():
     inner = ConditionSet(x, sin(x) + x > 0).intersect(Interval(-1, 1))
     outer = ConditionSet(x, Contains(x, inner), S.Reals)
     assert outer == ConditionSet(x, sin(x) + x > 0, Interval(-1, 1))
+
+    def test_duplicate():
+        from sympy.core.function import BadSignatureError
+        # test coverage for line 95 in conditionset.py, check for duplicates in symbols
+        dup = symbols('a,a')
+        raises(BadSignatureError, lambda: ConditionSet(dup, x < 0))
```
Comment (sylee957) [inline sympy/sets/tests/test_conditionset.py:275]:
```suggestion
def test_duplicate():
    from sympy.core.function import BadSignatureError
    # test coverage for line 95 in conditionset.py, check for duplicates in symbols
    dup = symbols('a,a')
    raises(BadSignatureError, lambda: ConditionSet(dup, x < 0))
```

File: sympy/sets/tests/test_ordinals.py [hunk 2]
```diff
@@ -54,3 +55,13 @@ def test_exponentiation():
...
+
+def test_comapre_not_instance():
+    w = OmegaPower(omega + 1 , 1)
+    assert(not (w == None))
+    assert(not (w < 5))
+    raises(TypeError, lambda: w < 6.66)
+
+def test_is_successort():
+    w = Ordinal(OmegaPower(5,1))
+    assert(not w.is_successor_ordinal)
```
Comment (sylee957) [inline sympy/sets/tests/test_ordinals.py:67]:
```suggestion
    assert not w.is_successor_ordinal
```

File: sympy/sets/tests/test_powerset.py [hunk 3]
```diff
@@ -128,3 +128,14 @@ def test_powerset_method():
     # Not finite sets
     A = Interval(0, 1)
     assert A.powerset() == PowerSet(A)
+
+def test_is_subset():
+    # covers line 101-102
+    # initalize powerset(1), which is a subset of powerset(1,2)
+    subset = PowerSet(FiniteSet(1))
+    pset = PowerSet(FiniteSet(1,2))
+    bad_set = PowerSet(FiniteSet(2,3))
```
Comment (sylee957) [inline sympy/sets/tests/test_powerset.py:137]:
```suggestion
    pset = PowerSet(FiniteSet(1, 2))
    bad_set = PowerSet(FiniteSet(2, 3))
```

File: sympy/sets/tests/test_ordinals.py [hunk 4]
```diff
@@ -54,3 +55,13 @@ def test_exponentiation():
...
     assert (w**w)*(w**w) == w**(w*2)
+
+def test_comapre_not_instance():
+    w = OmegaPower(omega + 1 , 1)
+    assert(not (w == None))
+    assert(not (w < 5))
+    raises(TypeError, lambda: w < 6.66)
+
+def test_is_successort():
+    w = Ordinal(OmegaPower(5,1))
```
Comment (sylee957) [inline sympy/sets/tests/test_ordinals.py:66]:
```suggestion
    w = Ordinal(OmegaPower(5, 1))
```

File: sympy/sets/tests/test_ordinals.py [hunk 5]
```diff
@@ -54,3 +55,13 @@ def test_exponentiation():
     assert w**3 == w*w*w
     assert w**(w + 1) == Ordinal(OmegaPower(omega + 1, 1))
     assert (w**w)*(w**w) == w**(w*2)
+
+def test_comapre_not_instance():
+    w = OmegaPower(omega + 1 , 1)
```
Comment (sylee957) [inline sympy/sets/tests/test_ordinals.py:60]:
```suggestion
    w = OmegaPower(omega + 1, 1)
```

File: sympy/sets/tests/test_contains.py [hunk 6]
```diff
@@ -40,3 +40,7 @@ def test_as_set():
     # containment in a Set -- rewriting as a Set is not yet implemented
     raises(NotImplementedError, lambda:
            Contains(x, FiniteSet(y)).as_set())
+
+def test_type_error():
+    # Pass in a parameter not of type "set"
+    raises(TypeError, lambda: Contains(2,None))
```
Comment (sylee957) [inline sympy/sets/tests/test_contains.py:46]:
```suggestion
    raises(TypeError, lambda: Contains(2, None))
```

**Comment by vijairam20** [comment]
@sylee957  I have committed the changes is it okay to be merged?

**Pull request merged**